### DISCUSSION TITLE
Add a button to accordion dropdown without triggering validateDOMNesting Error

### DISCUSSION THREAD

**Discussion by marco-braid** [chakra-ui/chakra-ui#5394] [General]
I'd like to add a button component to the Accordion dropdown. The accordion dropdown is implemented as a button so adding a button triggers a validteDOMNesting error. Are there any suggested workarounds to this?

**Comment by pgriger** [comment]
Kind of quick-and-dirty solution is to declare AccordionButton element as non-button element.

`<AccordionButton as='div'>`
`  <Box as="span" flex='1' textAlign='left'>Title</Box>`
`  <IconButton icon={<DeleteIcon />} onClick={deleteItem} />`
`</AccordionButton>`
